src/Controls/src/Core/Shell/ShellItem.cs

```diff
@@ -328,6 +327,6 @@ internal override void SendDisappearing()
 			}
 		}
 
-		IReadOnlyList<Maui.IVisualTreeElement> IVisualTreeElement.GetVisualChildren() => Items.ToList().AsReadOnly();
+		IReadOnlyList<Maui.IVisualTreeElement> IVisualTreeElement.GetVisualChildren() => Items.ToList();
```

Comment:
This avoids an allocation. `.ToList()` already creates a unique copy of the list, so we can just return it as-is.

https://source.dot.net/#System.Private.CoreLib/List.cs,246